src/client.js

```diff
@@ -357,9 +413,38 @@ export class StreamChat {
 		this.listeners[key] = this.listeners[key].filter(value => value !== callback);
 	}
 
+	_logApiRequest(type, url, data, config) {
+		this.logger('info', 'client:' + type + ' - Request - ' + url, {
+			tags: ['api', 'api_request', 'client'],
+			url,
+			payload: data,
+			config,
+		});
+	}
+
+	_logApiResponse(type, url, response) {
+		this.logger(
+			'info',
+			'client:' +
```

Comment:
@vishalnarkhede any reason not to use string interpolation? this looks like 💩

Reply:
Done!